packages/sign_in_with_apple/example/lib/main.dart

```diff
@@ -24,7 +24,16 @@ class _MyAppState extends State<MyApp> {
           child: Center(
             child: SignInWithAppleButton(
               onPressed: () async {
-                final credentials = await SignInWithApple.requestCredentials();
+                final credentials = await SignInWithApple.requestCredentials(
```

Comment:
Wondering whether we should make this simpler the API, so the Apple ID scopes can be directly set.

Don't see this changing a lot over time / gaining much configurability.

I think we could even handle the Keychain fallback with a `preferExistingKeychainEntry` parameter (diverging a little from the Apple-provided API). I think that makes it a lot simpler, is and actually nicer for the specific SiwA case (don't think people would try this library to _only_ get Keychain credentials).

Reply:
Yeah, let's discuss it in #45